app/controllers/base/ReportGeneratorWizardController.java

```
@@ -255,11 +255,6 @@ protected final String baseViewName() {
             .thenApply(offender -> storeReportFilename(params, offender))
             .thenApply(offender -> storeOffenderDetails(params, offender))
             .thenCompose(updatedParams -> generateAndStoreReport(wizardForm.bind(updatedParams).value().orElseGet(this::newWizardData)).
-                exceptionally(error -> {
```

This block was hiding errors (equivalent to a `catch` block with no throw)